gsdocs/2019/index.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
This file is under 2019, don't know if that's ok

Reply:
This is for the last year page

Reply:
Which did not have links to the projects anyway